pkg/controllers/cloud/cloudentityselector_controller.go

```diff
@@ -55,6 +55,12 @@ type CloudEntitySelectorReconciler struct {
 func (r *CloudEntitySelectorReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	_ = r.Log.WithValues("cloudentityselector", req.NamespacedName)
 
+	if !r.initialized {
+		if err := waitTillControllerInitialized(&r.initialized, initTimeout, CESController); err != nil {
```

Comment:
nit: waitTillControllerIsInitialized

Reply:
Done. 
This function still seems misplaced in controller_sync_status.go file.
Should we move it to individual controller?